.github/workflows/ci-images.yml

```diff
@@ -101,7 +101,7 @@ jobs:
 
     - name: "Build Image"
       env:
-        IMG_BAKE_ARGS: --load
+        IMG_BAKE_ARGS: --load --set '*.platform=linux/amd64'
```

Comment:
We explicitly install the qemu emulators, so this should technically work, even though it is merely user-space emulated. Did you verify this fails?

Reply:
it fails with 
```
ERROR: docker exporter does not currently support exporting manifest lists
```
CI run: https://github.com/osbuild/containers/runs/7955998333
Is there another way ?

Reply:
on further investigation, `bake` only succeeds and that builds the image but doesn't store the image anywhere.
`bake --load` fails because exporting is the problem (as in the message above), `bake --push` also succeeds, but this combines several steps into one, which is undesirable.
I believe the only option that fits in the current workflow easily is to use `bake --load` and `--set` to specify the platforms loaded from the hcl one by one.
What are your thoughts ?

Reply:
Updated CI as discussed.